stdlib/3/importlib/abc.pyi

```diff
@@ -91,7 +91,7 @@ if sys.version_info >= (3, 9):
         @abstractmethod
         def __truediv__(self, child: Traversable) -> Traversable: ...
         @abstractmethod
-        def open(self, mode: Literal["r", "rb"] = ..., *args: Any, **kwargs: Any) -> IO: ...
+        def open(self, mode: Literal["r", "rb"] = ..., *args: Any, **kwargs: Any) -> IO[AnyStr]: ...
```

Comment:
```suggestion
        def open(self, mode: Literal["r", "rb"] = ..., *args: Any, **kwargs: Any) -> IO[Any]: ...
```

(And remove the import.) As `AnyStr` is a type var without second use.

Reply:
Presumably it returns IO[str] for `r` and `IO[bytes]` for `rb`, so maybe we could use overloads? Perhaps that would be annoying in practice for an ABC though.

Reply:
@sritau, I thought I remembered from a previous discussion in typing-sig that we were generally OK with a single instance of a TypeVar if it had multiple constraints, as AnyStr does. But I'm fine changing this to Any if that's consistent with other typeshed stubs. 

@JelleZijlstra, yeah, I think it might be annoying to use an overload on an abstract method. I also don't know how all of the type checkers would behave on an abstract overloaded method. It's probably best to not overload it and allow subclasses to provide an overloaded version of the method.